task/07-yield-tasks.js

```diff
@@ -108,10 +133,16 @@ function* depthTraversalTree(root) {
  *
  */
 function* breadthTraversalTree(root) {
-    throw new Error('Not implemented');
+  let temp = [root]; // Name 'temp' conveys meaning of the variable as the temporary storage of some information in process of treatment.
```

Comment:
so it contains nodesToProcess ?

A rule of thumb - if you need a huge comment to describe why you chose this variable name then it's a bad name